tests/cmdline/commands/test_storage.py

```diff
@@ -169,11 +171,17 @@ def mock_maintain(*args, **kwargs):
 
     monkeypatch.setattr(storage, 'maintain', mock_maintain)
 
-    # Not passing user input should cause the command to exit without executing `storage.mantain` and so the last
+    raises = False
+    # Not passing user input should cause the command to exit
+    # without executing `storage.mantain` and so the last
     # message should be the prompt to continue or not.
-    result = run_cli_command(cmd_storage.storage_maintain, use_subprocess=False)
+    # In click>=8.2 this raises an exception which we need to catch.
```

Comment:
I don't fully understand this part. Do we do a `sys.exit` in this case and click now raises an error on a sys exit? Is this potentially an API break since the exit code changes?